app/src/main/res/drawable/ic_gender_24dp.xml

```diff
@@ -0,0 +1,16 @@
+<vector android:height="24dp" android:viewportHeight="512"
```

Comment:
Are we using standard drawable? Can you clarify where it's taken? 
The rest looks good to me.

Reply:
@miPlodder I didn't understood what are you talking about, but i just downloaded svg icon and imported vector asset in drawable. Should I change anything or is this ok?

Reply:
I'm just asking where this SVG has been taken from? (just share the link)

Reply:
[gender icon](https://www.flaticon.com/free-icon/toilet-signs_3917918?term=gender&page=1&position=58&related_item_id=3917918)
i edited it to match app theme